### ISSUE TITLE
exporter won't work because of different verion

### ISSUE THREAD

**Issue by MlerSu** [kabiiQ/BeatmapExporter#7]
![image](https://github.com/kabiiQ/BeatmapExporter/assets/111119321/dfae2977-58ad-4f71-a9f8-f5d66bce8f56)

I don't quite understand what this means, but since lazer did just update it might have something to do with it.

**Comment by kabiiQ** [comment]
You are correct about the lazer update, but I already had a release up a few hours ago that should work with the latest lazer: https://github.com/kabiiQ/BeatmapExporter/releases/tag/v1.3.0

It should display as the first line in the program when there is an update available. Most of the time if there is a BeatmapExporter update it is required to work with the latest version of lazer. 

That said, thanks for the screenshot. There should be a more friendly error message but the wording of that message changed so it stopped displaying.
